examples/with_winit/src/main.rs

```diff
@@ -49,10 +53,51 @@ struct Args {
     /// Switch between scenes with left and right arrow keys
     #[arg(long)]
     scene: Option<i32>,
+    #[arg(
+        long,
+        default_value_t = ClearColor::Black,
+        value_parser = PossibleValuesParser::new(["black", "white", "aquamarine", "crimson"])
+            .map(|s| s.parse::<ClearColor>().unwrap())
+    )]
+    clear_color: ClearColor,
```

Comment:
A scene which cycles through all clear colours of a specific luminance/saturation would be cool, although that seems equivalent to just doing the same thing with a normal fill.

Reply:
> This should be an Option, because scenes should be able to set the clear colour if not specified by the user.

Good idea! I made this an optional parameter and made it the responsibility of the examples to supply a default color.

> A scene which cycles through all clear colours of a specific luminance/saturation would be cool, although that seems equivalent to just doing the same thing with a normal fill.

I thought of this, though since I defined this as a render parameter (rather than a scene parameter) this would require changing the test_scenes plumbing a little bit so that render options can be modified and animated by the test scenes. Which is OK, and I'll think about it as a follow up.

I considered making this a scene parameter as well but I wasn't sure whether this belongs in the scene or as a renderer option. Maybe there is value in specifying a background color at the scene level; I don't feel strongly either way.

@dfrg @raphlinus Any thoughts?

Reply:
> Anything that can be parameterized wrt to scene probably should so I prefer the current state.

My proposal would be that the priority is:
1. Provided by user on command line
2. Provided by scene
3. Default for renderer (effectively always black)

The option in 2 is mostly useful for a theoretical future where we use `test_scenes` for e.g. CI tests.
In that case, we'd want to validate that base colours works correctly (e.g. correct colour space/alpha sent through, etc.). So a scene which changes the base colour would therefore make sense.

OTOH, the current state is also fine, as implementing this scene parameter in such a case would be trivial.

Reply:
I just pushed a new commit that adds a new scene parameter and a test scene that animates a CIE HCL color (`peniko::Color` had a constructor for this, so I just went with it). I implemented the precedence that you specified. Please try it out.

I'll wait for your approval before I merge the change.